index.js

```diff
@@ -111,6 +114,8 @@ async function run(context, plugins) {
 
       if (options.dryRun) {
         logger.warn(`Skip ${nextRelease.gitTag} tag creation in dry-run mode`);
+      } else if (options.disablePush) {
+        logger.warn(`Skip ${nextRelease.gitTag} tag creation in disable-push mode`);
```

Comment:
the version tag is very core to the operation of semantic-release. it is _very_ unlikely for us to accept a change that would enable disabling this functionality

Reply:
Answered in: https://github.com/semantic-release/semantic-release/pull/1924#issuecomment-835769990